android/app/src/main/java/app/organicmaps/MwmActivity.java

```diff
@@ -1264,6 +1266,16 @@ private boolean isFullscreen()
         Framework.nativeGetChoosePositionMode() == Framework.ChoosePositionMode.NONE;
   }
 
+  private void showFullscreenToastIfNeeded()
+  {
+    // Show the toast only 2 times so new behaviour doesn't confuse users
+    if (Config.longTapToastShowsCount(this) < 2)
+    {
+      Toast.makeText(this, R.string.long_tap_toast, Toast.LENGTH_LONG).show();
+      Config.incLongTapToastShowsCount(this);
+    }
```

Comment:
After some testing, a one-time long-length toast looks good enough. Two toasts look more annoying, it feels like the toast will always be displayed on entering fullscreen. So let's do it only once (use a bool).
```suggestion
    // Show the toast only 2 times so new behaviour doesn't confuse users
    if (Config.longTapToastShowsCount(this) < 2)
    {
      Toast.makeText(this, R.string.long_tap_toast, Toast.LENGTH_LONG).show();
      Config.incLongTapToastShowsCount(this);
    }
```

@kirylkaveryn can you please add commits to also show a one-time toast on iOS (on a separate branch, so @strump can then cherry-pick your iOS commits)?

Reply:
Done!